lib/filestorage.js

```diff
@@ -96,12 +100,12 @@ class FileStorage {
   //   cb - <Function>, callback
   //     err - <error>
   rm(id, cb) {
-    const file = path.join(this.dir, common.idToPath(id) + '.' + FS_EXT);
+    const file = this._makeFileExt(path.join(this.dir, common.idToPath(id)));
     fs.unlink(file, cb);
   }
 
   _compress(id, compression, cb) {
-    const file = path.join(this.dir, common.idToPath(id) + '.' + FS_EXT);
+    const file = this._makeFileExt(path.join(this.dir, common.idToPath(id)));
```

Comment:
We have
```js
    const file = this._makeFileExt(path.join(this.dir, ...chunks));
```
where extension should be added only after resolving path and I'd like to have persistent code style across the file and since this doesn't impact performance I decided to do it like this.